Source/WarpX.cpp

```diff
@@ -3049,13 +2966,15 @@ WarpX::ComputeDivE(amrex::MultiFab& divE, const int lev)
 {
     if ( WarpX::electromagnetic_solver_id == ElectromagneticSolverAlgo::PSATD ) {
 #ifdef WARPX_USE_FFT
-        spectral_solver_fp[lev]->ComputeSpectralDivE( lev, Efield_aux[lev], divE );
+        ablastr::fields::VectorField Efield_aux_lev = m_fields.get_alldirs("Efield_aux", lev);
+        spectral_solver_fp[lev]->ComputeSpectralDivE(lev, Efield_aux_lev, divE);
 #else
         WARPX_ABORT_WITH_MESSAGE(
             "ComputeDivE: PSATD requested but not compiled");
 #endif
     } else {
-        m_fdtd_solver_fp[lev]->ComputeDivE( Efield_aux[lev], divE );
+        ablastr::fields::VectorField Efield_aux_lev = m_fields.get_alldirs("Efield_aux", lev);
```

Comment:
Clang-tidy error "variable 'Efield_aux_lev' of type 'ablastr::fields::VectorField' (aka 'array<amrex::MultiFab *, 3>') can be declared 'const' [misc-const-correctness,-warnings-as-errors]":
```suggestion
        const ablastr::fields::VectorField Efield_aux_lev = m_fields.get_alldirs("Efield_aux", lev);
```